test/test_extended_models.py

```diff
@@ -79,20 +79,32 @@ def test_naming_conventions(model_fn):
 )
 @run_if_test_with_extended
 def test_schema_meta_validation(model_fn):
-    # TODO: add list of permitted fields
-    classification_fields = ["categories", "acc@1", "acc@5"]
+    # list of all possible supported high-level fields for weights meta-data
```

Comment:
Here we store all permitted high-level fields on the meta-data. Passing anything other than this will cause failures. This aims to keep the number of permitted fields low but we are able to add more fields by editing this list.